website/content/en/status/report-2023-07-2023-09/dpaa2.adoc

```diff
@@ -0,0 +1,44 @@
+=== NXP DPAA2 support
+
+Links +
+link:https://cgit.freebsd.org/src/tree/sys/dev/dpaa2[DPAA2 in the FreeBSD source tree] +
+link:https://github.com/mcusim/freebsd-src[DPAA2 on Github]
+
+Contact: Dmitry Salychev <dsl@FreeBSD.org> +
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+==== What is DPAA2?
+
+DPAA2 is a hardware-level networking architecture found in some NXP SoCs which contain hardware blocks including Management Complex (MC, a command interface to manipulate DPAA2 objects), Wire Rate I/O processor (WRIOP, packets distribution, queuing, drop decisions),
+Queues and Buffers Manager (QBMan, Rx/Tx queues control, Rx buffer pools) and the others. The Management Complex runs NXP-supplied firmware which provides DPAA2 objects as an abstraction layer over those blocks to simplify an access to the underlying hardware.
```

Comment:
```suggestion
DPAA2 is a hardware-level networking architecture found in some NXP SoCs which contain hardware blocks including Management Complex (MC, a command interface to manipulate DPAA2 objects), Wire Rate I/O processor (WRIOP, packets distribution, queuing, drop decisions), Queues and Buffers Manager (QBMan, Rx/Tx queues control, Rx buffer pools) and others.
The Management Complex runs NXP-supplied firmware which provides DPAA2 objects as an abstraction layer over those blocks to simplify access to the underlying hardware.
```

Reply:
> … blocks including … Management Complex …, Queues and Buffers Manager (…) and others.

That's _including_, not _comprising_, so you can lose the two words at the tail of the sentence: 

> … blocks including … Management Complex …, and Queues and Buffers Manager (…).